components/CatModal/CatModalView.tsx

```diff
@@ -33,134 +36,162 @@ const useStyles = createStyles((theme) => ({
 
 type SterializeStatus = 'sterilized' | 'unsterilized';
 
-export default function CatModalView({ cat, modal }: { cat?: Cat; modal: ModalConfig }) {
-  const theme = useMantineTheme();
+export interface FormValues {
+  gender: Gender;
+  observations: string;
+  mediaLinks: string[];
+  sterilizedStatus: SterializeStatus;
+  volunteerName?: string;
+  hospitalizationDate?: Date;
+  releaseDate?: Date;
+}
+
+export default function CatModalView({
+  cat,
+  modal,
+  setModal,
+}: {
+  cat: Cat | undefined;
+  modal: ModalConfig;
+  setModal: (modal: ModalConfig | undefined) => void;
+  zoneId: string;
+}) {
   const { classes } = useStyles();
-  const [statusSterialize, setStatusSterialize] = useState<SterializeStatus>('sterilized');
-  const [form, setForm] = useState<Cat>(
-    cat || {
-      gender: Gender.FEMALE,
-      observations:
-        'Enter here any observations, notes, that might help identify the cat more accurately.',
-      mediaLinks: [],
-    }
-  );
-  const [sterilizedForm, setSterilizedForm] = useState({
-    volunteerName: (cat as SterilizedCat)?.volunteerName || '',
-    hospitalizationDate: (cat as SterilizedCat)?.hospitalizationDate || '',
-    releaseDate: (cat as SterilizedCat)?.releaseDate || '',
+  const theme = useMantineTheme();
+  const form = useForm<FormValues>({
+    initialValues: {
+      ...(cat || {
+        gender: Gender.UNKNOWN,
+        observations:
+          'Enter here any observations, notes, that might help identify the cat more accurately.',
+        mediaLinks: [],
+      }),
```

Comment:
Not sure if it is an improvement, but I'd write the code as follows
`
initialValues: {
        gender: Gender.UNKNOWN,
        observations: 'Enter here any observations, notes, that might help identify the cat more accurately.',
        mediaLinks: [],
        ...(cat ?? {}),
        volunteerName: .........
`